DataFormats/FTLRecHit/interface/FTLUncalibratedRecHit.h

```diff
@@ -0,0 +1,67 @@
+#ifndef DATAFORMATS_FTLUNCALIBRATEDRECHIT
+#define DATAFORMATS_FTLUNCALIBRATEDRECHIT
+
+#include <vector>
+#include "DataFormats/DetId/interface/DetId.h"
+
+class FTLUncalibratedRecHit {
+
+  public:
+  
+  typedef DetId key_type;
+
+  enum Flags {
+          kGood=-1,                 // channel is good (mutually exclusive with other states)  setFlagBit(kGood) reset flags_ to zero 
+          kPoorReco,                // channel has been badly reconstructed (e.g. bad shape, bad chi2 etc.)
+          kSaturated,               // saturated channel
+          kOutOfTime               // channel out of time
```

Comment:
I do not find these used anywhere in this PR.
Do you expect the flags to follow this naming convention from some first principles?
Maybe it's better to introduce these enums when they are useful